tests/container_disk_test.go

```diff
@@ -62,8 +62,14 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	})
 
 	DescribeTable("should", func(image string, policy k8sv1.PullPolicy, expectedPolicy k8sv1.PullPolicy) {
-		vmi := tests.NewRandomVMIWithEphemeralDisk(image)
+		vmi := libvmi.New(libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+			libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			libvmi.WithNamespace(testsuite.GetTestNamespace(nil)),
```

Comment:
Can we just use `NamespaceTestDefault` variable directly, instead of passing `nil` argument to this function that will return `NamespaceTestDefault` (?)

Reply:
done. thanks @codingben